script/libraries/Constants.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8;
 
 library Constants {
-    bytes32 constant SALT = bytes32(uint256(26));
+    bytes32 constant SALT = bytes32(uint256(28));
```

Comment:
maybe i'm being stupid, but I thought the salt was meant to stay the same? ;o

Reply:
It changes as we re-deploy. If we use the same one we get a collision.